### ISSUE TITLE
Localization for names

### ISSUE THREAD

**Issue by gaboprieto** [altafc22/csc_picker#35]
I would like to be able to have the country names in the locale that I want, like if the local is 'es', then "United States" should be "Estados Unidos", and so on.

Or is there any way to achieve this right now it would be great!

**Comment by davidalonsobadia** [comment]
Yep, I agree. This is an important feature that this great package is missing.
